packages/sit-onyx/src/i18n/locales/en-US.json

```
@@ -4,6 +4,13 @@
   "apply": "Apply",
   "back": "Back",
   "close": "Close",
+  "data-grid": {
```

```suggestion
  "dataGrid": {
```

We usually use camelCase for translation keys :)